extras/physfssdl3.c

```diff
@@ -24,6 +24,7 @@
 /* This works with SDL3. For SDL1 and SDL2, use physfsrwops.h */
 
 #include "physfssdl3.h"
+#include "SDL3/SDL_iostream.h"
```

Comment:
Pretty sure individual headers should not be included.
Also `physfssdl3.h` already includes `<SDL3/SDL.h>`

Reply:
Not sure how that snuck in there. Maybe VSCode clang plugin being a bit too enthusiastic